conf/messages.en

```diff
@@ -71,9 +71,9 @@ parts.cacheGroup.none=None yet
 parts.section.alertMail=Alert mail settings
 parts.alertMail.condition=Condition
 parts.alertMail.condition.label=Trigger condition
-parts.alertMail.condition.absolute={0} 
+parts.alertMail.condition.absolute={0}
 parts.alertMail.condition.relative={0}%
-parts.alertMail.condition.or= or 
```

Comment:
Nah, it actually doesn't. It's currently setting inside a div that ends up removing all white space anyway.